pkg/project/registry/rest.go

```diff
@@ -0,0 +1,68 @@
+package registry
```

Comment:
Aligning with upstream pattern:

https://github.com/GoogleCloudPlatform/kubernetes/tree/master/pkg/registry/namespace

Note the strategy as I understand is intended to be shared across different persistence options (etcd / proxy / etc.)